cognite_toolkit/_cdf_tk/commands/auth.py

```diff
@@ -209,7 +209,7 @@ def verify(
         if cdf_toolkit_group is None:
             return VerifyAuthResult()
 
-        if not is_demo and not is_user_in_toolkit_group:
+        if not is_demo and not is_user_in_toolkit_group and cdf_toolkit_group.source_id:
```

Comment:
This assume that your current credentials is the one that should be used for running Toolkit. This might not be true.
```suggestion
        if not is_demo and not is_user_in_toolkit_group:
```